src/test/java/seedu/address/model/person/TutorTest.java

```diff
@@ -52,40 +53,42 @@ public void isSamePerson() {
 
     @Test
     public void equals() {
+        String alternateQualification = "1";
```

Comment:
May want to declare a static final constant in `CommandTestUtil` instead (might be useful for future tests too)